surface_analyses/ele_patches.py

```diff
@@ -93,6 +93,18 @@ def main(argv=None):
         action='store_true',
         help='For an antibody Fv region as input: check whether patches belong to CDRs.',
     )
+    parser.add_argument(
+        '-n','--n_patches',
+        type=int,
+        default=0,
+        help='Restrict output to n patches. Positive values output n largest patches, negative n smallest patches.',
+    )
+    parser.add_argument(
+        '-s','--size_cutoff',
```

Comment:
Is it a good idea to have the short options? We have many options for our script, and many of them don't have short options.